platform/core/src/studies/services/wado/retrieveMetadata.js

```diff
@@ -355,6 +355,32 @@ async function makeSOPInstance(server, study, instance) {
     lossyImageCompressionMethod: DICOMWeb.getString(instance['00282114']),
     echoNumber: DICOMWeb.getString(instance['00180086']),
     contrastBolusAgent: DICOMWeb.getString(instance['00180010']),
+    sort4DSeriesTags: {
```

Comment:
Can we take these out of this sub object here in the instance and then organize them into the sort4DSeriesTags object in the is4DSeries function which actually does the sorting?